tools/test-backend

```diff
@@ -65,6 +65,27 @@ if __name__ == "__main__":
                       default=False,
                       help="Show which tests are slowest.")
 
+    from version import PROVISION_VERSION
+    version_file = 'var/provision_version'
+    try:
+        version = open(version_file).read().strip()
+    except:
+        print('''
+            You must populate %s by running
+            provision.py. Or, if you know you have provisioned
+            recently, run this command:
+
+            echo %s > %s''' % (version_file, PROVISION_VERSION, version_file))
+        sys.exit(1)
+
+    if version != PROVISION_VERSION:
```

Comment:
we may want to do a 1.2 type of version number, where if you're lower but at the same major version number we don't complain?